### PR TITLE
splat-1897 Add to contribution guidelines

### PR DESCRIPTION
**Pull request by adamdockter-wk**

### CODE REVIEW

**Comment by aviary3-wk** [comment]
## Security Insights
<details>
<summary>(5) Vulnerable direct dependencies were detected</summary>
<li>1 vulns in <code>aiohttp < 3.7.4</code> via <code>lib/python/requirements_dev_asyncio.txt</code></li>
<li>1 vulns in <code>org.apache.thrift:libthrift < 0.14.0</code> via <code>lib/java/pom.xml</code></li>
<li>1 vulns in <code>org.apache.thrift:libthrift < 0.14.0</code> via <code>test/integration/java/frugal-integration-test/pom.xml</code></li>
<li>1 vulns in <code>org.nanohttpd:nanohttpd</code> via <code>test/integration/java/frugal-integration-test/pom.xml</code> with no fix reported by GitHub</li>
<li>2 vulns in <code>github.com/nats-io/nats-server/v2 < 2.7.4</code> via <code>lib/go/go.mod</code></li>
</details>

## Action Items
* Review dependencies for available updates
* See <a href="https://splunk.workiva.net/en-US/app/search/security_vulnerable_dependencies?form.repository=frugal">this Splunk dashboard</a> for more CVE details
* Review PR for [security impact](https://wiki.atl.workiva.net/display/SECURITY/Development+Security+Review+Guidelines); comment "security review required" if needed or unsure
* Verify [`aviary.yaml`](https://wiki.atl.workiva.net/pages/viewpage.action?pageId=42762752#SecurityInsightFramework&Tooling-ConfiguringAviary) coverage of security relevant code
---
_Questions or Comments? Reach out on Slack: #support-infosec._

**Review by amitjain-wk — APPROVED** [review]
+1 LGTM

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,6 +1,10 @@
 Contributing to Frugal
 =======================
 
+## Before you start
+
+Opening a PR doesn't guarantee success of future merging, it's in your best interest to message the repo owners ahead of time to discuss a feature request. All feature will be evaluated based on their usefulness within the community of user.
```
Comment (toddbeckman-wf) [inline CONTRIBUTING.md:6]:
```suggestion
Opening a PR doesn't guarantee a feature will be merged. Please discuss with the repo owners ahead of time to discuss a feature request. All features will be evaluated based on their usefulness within the community of users.
```

**Review by toddbeckman-wf — APPROVED** [review]

**Comment by adamdockter-wk** [comment]
@Workiva/release-management-p

**Review by rmconsole-wf — APPROVED** [review]
+1 from RM

**Pull request merged**